### PR TITLE
Add nsqd config examples

### PR DESCRIPTION
**Pull request by harlow**
* Add documentation for nsqd config endpoints

Minor updates:

* When trying to run jekyll I ran into this deprecation so I also updated the `config.yml`

screen shot 2016-01-22 at 11 56 51 pm

### CODE REVIEW

File: _posts/2012-12-01-nsqd.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ can optionally listen on a third port for HTTPS.
  * [`/stats`](#get-stats) - comprehensive runtime telemetry
  * [`/pub`](#post-pub) - publish a message to a topic
  * [`/mpub`](#post-mpub) - publish multiple messages to a topic
+ * [`/config/nsqlookupd_tcp_addresses`](#get-config) - returns array of nsqlookupd  addresses
+ * [`/config/nsqlookupd_tcp_addresses`](#put-config) - updates nsqlookupd TCP addresses
```
Comment (mreiferson) [inline _posts/2012-12-01-nsqd.md]: do we need both entries here?  should we combine to one?

Comment (harlow) [inline _posts/2012-12-01-nsqd.md]: Agreed. I think one link to the config section makes sense. Updated in 6c7f2d9

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
```
Comment (harlow) [inline Gemfile]: @mreiferson I'm not sure what your stance is on having a Gemfile. And maybe it warrants a PR outside of this one. 
https://help.github.com/articles/using-jekyll-with-pages/#installing-jekyll

File: README.md [hunk 3]
```diff
@@ -7,8 +7,8 @@ http://nsq.io/
 Offline documentation requires [jekyll][jekyll]:
 
 ```bash
-$ gem install jekyll
-$ jekyll serve --safe --baseurl ''
+$ bundle install
+$ bundle exec jekyll serve --safe --watch --baseurl ''
```
Comment (harlow) [inline README.md]: Jekyll is running for me now, but when i visit http://0.0.0.0:4000/ it's redirecting to http://nsq.io//

Comment (harlow) [inline README.md]: changed to 127.0.0.1 and works fine for me now!

**Comment by mreiferson** [comment]
LGTM thanks @harlow

**Pull request merged**